teos/src/main.rs

```diff
@@ -265,11 +266,33 @@ async fn main() {
         internal_rpc_api_uri,
         shutdown_signal_http,
     ));
+
+    // Add Tor Onion Service for public API
+    log::info!("above tor_support");
+    log::info!("tor_support: {}", conf.tor_support);
+    let mut tor_task: Option<tokio::task::JoinHandle<()>> = Option::None;
```

Comment:
Forgot this one comment:

Type annotation is not really needed here, it can be inferred by the context.

```suggestion
     let mut tor_task = None;
```